src/auth/SignUp.tsx

```diff
@@ -63,79 +57,73 @@ const SignUp = () => {
+              <AlertTitle>Error!</AlertTitle>
+              <AlertDescription>{error}</AlertDescription>
+            </Alert>
+          )}
+          <form onSubmit={handleSubmit(onSubmit)}>
+            <Input
+              fontSize="1rem"
+              height="3.5rem"
+              size="lg"
+              name="email"
+              control={control}
+              errors={errors}
+              placeholder="Email"
+              _placeholder={{ opacity: 1, color: 'whiteAlpha.700' }}
+              backgroundColor={'#21242F'}
+            />
+            <Input
+              marginY={2}
+              size="lg"
+              fontSize="1rem"
+              type="password"
+              height="3.5rem"
+              name="password"
+              placeholder="Password"
+              _placeholder={{ opacity: 1, color: 'whiteAlpha.700' }}
+              control={control}
+              errors={errors}
+              backgroundColor={'#21242F'}
+            />
 
-              <form onSubmit={handleSubmit(onSubmit)}>
-                <Input
-                  size="lg"
-                  name="email"
-                  label="Email"
-                  control={control}
-                  errors={errors}
-                />
-                <InputGroup size="md" marginY={5}>
-                  <Input
-                    // paddingRight="4.5rem"
-                    size="lg"
-                    name="password"
-                    type={show ? 'text' : 'password'}
-                    placeholder="Enter password"
-                    control={control}
-                    errors={errors}
-                  />
-                  <InputRightElement width="4.5rem">
-                    <Button h="1.75rem" size="sm" onClick={handleClick}>
-                      {show ? 'Hide' : 'Show'}
-                    </Button>
-                  </InputRightElement>
-                </InputGroup>
-                <InputGroup size="md">
-                  <Input
-                    size="lg"
-                    // paddingRight="4.5rem"
-                    name="passwordConfirm"
-                    type={show ? 'text' : 'password'}
-                    placeholder="Confirm password"
-                    control={control}
-                    errors={errors}
-                  />
-                  <InputRightElement width="4.5rem">
-                    <Button h="1.75rem" size="sm" onClick={handleClick}>
-                      {show ? 'Hide' : 'Show'}
-                    </Button>
-                  </InputRightElement>
-                </InputGroup>
+            <Input
+              size="lg"
+              fontSize="1rem"
+              type="password"
+              height="3.5rem"
+              name="confirmPassword"
+              placeholder="Confirm Password"
+              _placeholder={{ opacity: 1, color: 'whiteAlpha.700' }}
+              control={control}
+              errors={errors}
+              backgroundColor={'#21242F'}
```

Comment:
Please move the colours into the useCustomColours file or use the nearest Chakra equivalent @murvyn